src/nile/cli.py

```diff
@@ -2,7 +2,7 @@
 """Nile CLI entry point."""
 import logging
 
-import click
+import asyncclick as click
```

Comment:
Using asyncclick will add the requirement for plugins as well?

Reply:
not sure @andrew-fleming